network/p2p/libp2pNode.go

```diff
@@ -55,6 +56,10 @@ func DefaultLibP2PNodeFactory(ctx context.Context, log zerolog.Logger, me flow.I
 
 	connGater := NewConnGater(log)
 
+	//resolver, err := dns.NewResolver(metrics)
+	//if err != nil {
+	//	return nil, fmt.Errorf("could not create dns resolver: %w", err)
+	//}
```

Comment:
Commented out this temporarily since we need benchmarking results on DNS cache to activate it.